tests/test_hull.py

```
@@ -33,29 +6,25 @@
+A simple right triangle to start with
 - should have 3 norms from 3 equations as a starter
 """
 
 
-def test_hull():
+def test_hull() -> None:
     """Basic build test, just see if it can be built and it rejects bad dimension inputs"""
-    hull = Hull(pts)
-    assert hull.cv_hull
+    hull = Hull(pts)  # noqa F841
     with pytest.raises(ValueError):
```

Ruff (0.14.4)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)